db/schema.sqlite.sql

```diff
@@ -1,3 +0,0 @@
--- Leaving this file empty forces the whole migration
--- and fixes a bug when using Typo installer.
--- Please, leave them as is.
```

Comment:
Hmmm seems like these files say "leave them as is" so maybe you don't want to delete them afterall?